Pages/Hotels/hotelsHome.js

```diff
@@ -0,0 +1,481 @@
+    const allHandles = await browser.getWindowHandles();
+    let titleContains = 'DirectWord';
+    let handle = ''
+    for (handle of allHandles) {
+        await browser.switchToWindow(handle);
+        const pageTitle = await browser.getTitle();
+        if (!pageTitle.includes(titleContains)) {
+            await browser.closeWindow();
+        }
+    }
+}
+
+
+async switchToListTab() {
+    const allHandles = await browser.getWindowHandles();
+    let titleContains = 'Property Info';
+    let handle = ''
+    for (handle of allHandles) {
+        await browser.switchToWindow(handle);
+        const pageTitle = await browser.getTitle();
+        if (!pageTitle.includes(titleContains)) {
+            await browser.closeWindow();
+        }
+    }
+}
+
+async clickSignInButton() {
+    await this.commands.clickWebElement(this.signInButton);
+}
+
+async clickSignInLink() {
+    await this.commands.clickWebElement(this.signInLink);
+}
+
+async clickSignUpLink() {
+    await this.commands.clickWebElement(this.signUpLink);
+}
+
+async clickFeedbackLink() {
+    await this.commands.clickWebElement(this.feedbackLink);
+}
+
+async clickTraveler() {
+    await $(this.travelersLocator).waitForDisplayed();
+    await this.commands.clickWebElement(this.travelersLocator);
+}
+
+async clickAdultsIncrease() {
+    await this.commands.clickWebElement(this.adultsIncrease);
+}
+
+async clickAdultsDecrease() {
+    await this.commands.clickWebElement(this.adultsDecrease);
+}
+
+async setNumOfAdults(num){ 
+    for(let i=2; i<num ; i++){
+        await this.commands.clickWebElement(this.addAdultsButton);
+        const numOfAdults = await this.commands.getTextFromWebElement(this.numOfAdults);
+        if(numOfAdults.localeCompare(num)===0){
+            break;
+        }
+    }
+}
+async setNumOfChildren(num){
+    for(let i=0; i<num ; i++){
+        await this.commands.clickWebElement(this.addChildrenButton);
+        const numOfChildren = await this.commands.getTextFromWebElement(this.numOfChildren);
+        if(numOfChildren.localeCompare(num)===0){
+            break;
+        }
+    }
+}
+
+async clickChildMinusButton(num){
+    for(let i=0; i<num;i++){
+        await this.commands.clickWebElement(this.removeChildrenButton);
+    }
+}
+
+async clickChildIncrease() {
+    await this.commands.clickWebElement(this.childIncrease);
+}
+
+async clickChildDecrease() {
+    await this.commands.clickWebElement(this.childDecrease);
+}
+
+
+async selectChild1Age(num){
+    await this.commands.selectFromDropdown(this.child1Age, num);
+}
+
+async selectChild2Age(num) {
+    await this.commands.selectFromDropdown(this.child2Age, num);
+}
+
+async selectChild3Age(num) {
+    await this.commands.selectFromDropdown(this.child3Age, num);
+}
+
+async clickDoneButton() {
+    await this.commands.clickWebElement(this.travelersDoneButton);
+}
+
+async travelersText () {
+    let text = await command.getTextFunction(this.travelers)
+    console.log('RIGHT HERE' + text);
```

Comment:
You can log stuffs for local debugging purpose but remove them before pushing to repo.